### ISSUE TITLE
Upgrade CodeMirror to latest version

### ISSUE THREAD

**Issue by microbit-robert** [microbit-foundation/python-editor-next#559]
Upgrade CodeMirror to latest version and test/check for any negative side effects/errors. Note to check indentation specifically.

**Comment by microbit-matt-hillsdon** [comment]
Blocked by https://github.com/codemirror/codemirror.next/issues/722
Perhaps we can work around by poking the editor after the font change.

**Comment by microbit-matt-hillsdon** [comment]
New issue to investigate. Not tried reproducing on main yet:

https://user-images.githubusercontent.com/44397098/153711380-c541a012-80f2-4052-a698-d409e40ef3d7.mp4
